[CI] EsqlClientYamlIT class failing

**Build Scans:**
- [elasticsearch-periodic #5454 / openjdk21_checkpart3_java-fips-matrix](https://gradle-enterprise.elastic.co/s/ua4pvydvyzkry)
- [elasticsearch-pull-request #47712 / part-3](https://gradle-enterprise.elastic.co/s/jnbtl7l7yrjy6)
- [elasticsearch-pull-request #47342 / part-3](https://gradle-enterprise.elastic.co/s/u6a66ybboo6v4)
- [elasticsearch-pull-request #47008 / part-3](https://gradle-enterprise.elastic.co/s/wphzhvbde7nwq)
- [elasticsearch-pull-request #46899 / part-3](https://gradle-enterprise.elastic.co/s/dh6pnojbpyqi6)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:esql:qa:server:multi-node:yamlRestTest" --tests "org.elasticsearch.xpack.esql.qa.multi_node.EsqlClientYamlIT.test {p0=esql/40_tsdb/from index pattern explicit counter use}" -Dtests.seed=7CE2D90B6E046238 -Dtests.locale=sr-Cyrl-XK -Dtests.timezone=Europe/Moscow -Druntime.java=21 -Dtests.fips.enabled=true
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.esql.qa.multi_node.EsqlClientYamlIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!(org.elasticsearch.xpack.esql.qa.multi_node.EsqlClientYamlIT),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
org.elasticsearch.client.ResponseException: method [GET], host [http://127.0.0.1:37745], URI [/_nodes/stats], status line [HTTP/1.1 503 Service Unavailable]
{"error":{"root_cause":[{"type":"master_not_discovered_exception","reason":null}],"type":"master_not_discovered_exception","reason":null},"status":503}
```

**Issue Reasons:**
- [main] 5 failures in class org.elasticsearch.xpack.esql.qa.multi_node.EsqlClientYamlIT (0.8% fail rate in 593 executions)
- [main] 4 failures in step part-3 (1.1% fail rate in 349 executions)
- [main] 4 failures in pipeline elasticsearch-pull-request (1.2% fail rate in 346 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

Looks like an unknown writeable tripped an assertion that killed a node in the multi node test suite:


```
[2024-12-19T16:33:34,469][ERROR][o.e.b.ElasticsearchUncaughtExceptionHandler] [test-cluster-1] fatal error in thread [elasticsearch-error-rethrower], exiting
java.lang.AssertionError: Unknown NamedWriteable [org.elasticsearch.compute.operator.Operator$Status][lookup]
	at org.elasticsearch.common.io.stream.NamedWriteableRegistry.throwOnUnknownWritable(NamedWriteableRegistry.java:150) ~[elasticsearch-9.0.0-SNAPSHOT.jar:?]
	at org.elasticsearch.common.io.stream.NamedWriteableRegistry.getReader(NamedWriteableRegistry.java:126) ~[elasticsearch-9.0.0-SNAPSHOT.jar:?]
	at org.elasticsearch.common.io.stream.NamedWriteableRegistry.getReader(NamedWriteableRegistry.java:109) ~[elasticsearch-9.0.0-SNAPSHOT.jar:?]
	at org.elasticsearch.common.io.stream.NamedWriteableAwareStreamInput.readNamedWriteable(NamedWriteableAwareStreamInput.java:56) ~[elasticsearch-9.0.0-SNAPSHOT.jar:?]
	at org.elasticsearch.common.io.stream.NamedWriteableAwareStreamInput.readNamedWriteable(NamedWriteableAwareStreamInput.java:33) ~[elasticsearch-9.0.0-SNAPSHOT.jar:?]
	at org.elasticsearch.common.io.stream.StreamInput.readOptionalNamedWriteable(StreamInput.java:1178) ~[elasticsearch-9.0.0-SNAPSHOT.jar:?]
	at org.elasticsearch.compute.operator.DriverStatus$OperatorStatus.<init>(DriverStatus.java:285) ~[?:?]
	at org.elasticsearch.common.io.stream.StreamInput.readCollectionAsImmutableList(StreamInput.java:1204) ~[elasticsearch-9.0.0-SNAPSHOT.jar:?]
	at org.elasticsearch.compute.operator.DriverStatus.<init>(DriverStatus.java:114) ~[?:?]
	at org.elasticsearch.common.io.stream.NamedWriteableAwareStreamInput.readNamedWriteable(NamedWriteableAwareStreamInput.java:57) ~[elasticsearch-9.0.0-SNAPSHOT.jar:?]
	at org.elasticsearch.common.io.stream.NamedWriteableAwareStreamInput.readNamedWriteable(NamedWriteableAwareStreamInput.java:33) ~[elasticsearch-9.0.0-SNAPSHOT.jar:?]
	at org.elasticsearch.common.io.stream.StreamInput.readOptionalNamedWriteable(StreamInput.java:1178) ~[elasticsearch-9.0.0-SNAPSHOT.jar:?]
	at org.elasticsearch.tasks.TaskInfo.from(TaskInfo.java:74) ~[elasticsearch-9.0.0-SNAPSHOT.jar:?]
	at org.elasticsearch.action.support.tasks.TransportTasksAction$NodeTasksResponse.<init>(TransportTasksAction.java:337) ~[elasticsearch-9.0.0-SNAPSHOT.jar:?]
	at org.elasticsearch.action.support.tasks.TransportTasksAction.lambda$new$1(TransportTasksAction.java:155) ~[elasticsearch-9.0.0-SNAPSHOT.jar:?]
	at org.elasticsearch.action.ActionListenerResponseHandler.read(ActionListenerResponseHandler.java:64) ~[elasticsearch-9.0.0-SNAPSHOT.jar:?]
	at org.elasticsearch.action.ActionListenerResponseHandler.read(ActionListenerResponseHandler.java:26) ~[elasticsearch-9.0.0-SNAPSHOT.jar:?]
	at org.elasticsearch.transport.TransportService$UnregisterChildTransportResponseHandler.read(TransportService.java:1786) ~[elasticsearch-9.0.0-SNAPSHOT.jar:?]
	at org.elasticsearch.transport.TransportService$UnregisterChildTransportResponseHandler.read(TransportService.java:1755) ~[elasticsearch-9.0.0-SNAPSHOT.jar:?]
	at org.elasticsearch.transport.TransportService$ContextRestoreResponseHandler.read(TransportService.java:1490) ~[elasticsearch-9.0.0-SNAPSHOT.jar:?]
	at org.elasticsearch.transport.TransportService$ContextRestoreResponseHandler.read(TransportService.java:1477) ~[elasticsearch-9.0.0-SNAPSHOT.jar:?]
	at org.elasticsearch.transport.InboundHandler.doHandleResponse(InboundHandler.java:420) ~[elasticsearch-9.0.0-SNAPSHOT.jar:?]
	at org.elasticsearch.transport.InboundHandler.handleResponse(InboundHandler.java:383) ~[elasticsearch-9.0.0-SNAPSHOT.jar:?]
	at org.elasticsearch.transport.InboundHandler.executeResponseHandler(InboundHandler.java:150) ~[elasticsearch-9.0.0-SNAPSHOT.jar:?]
	at org.elasticsearch.transport.InboundHandler.messageReceived(InboundHandler.java:125) ~[elasticsearch-9.0.0-SNAPSHOT.jar:?]
	at org.elasticsearch.transport.InboundHandler.inboundMessage(InboundHandler.java:98) ~[elasticsearch-9.0.0-SNAPSHOT.jar:?]
	at org.elasticsearch.transport.TcpTransport.inboundMessage(TcpTransport.java:822) ~[elasticsearch-9.0.0-SNAPSHOT.jar:?]
	at org.elasticsearch.transport.InboundPipeline.forwardFragments(InboundPipeline.java:125) ~[elasticsearch-9.0.0-SNAPSHOT.jar:?]
	at org.elasticsearch.transport.InboundPipeline.doHandleBytes(InboundPipeline.java:97) ~[elasticsearch-9.0.0-SNAPSHOT.jar:?]
	at org.elasticsearch.transport.InboundPipeline.handleBytes(InboundPipeline.java:62) ~[elasticsearch-9.0.0-SNAPSHOT.jar:?]
	at org.elasticsearch.transport.netty4.Netty4MessageInboundHandler.channelRead(Netty4MessageInboundHandler.java:55) ~[?:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444) ~[?:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420) ~[?:?]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412) ~[?:?]
	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:107) ~[?:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444) ~[?:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420) ~[?:?]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412) ~[?:?]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1357) ~[?:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:440) ~[?:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420) ~[?:?]
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:868) ~[?:?]
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:166) ~[?:?]
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788) ~[?:?]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysPlain(NioEventLoop.java:689) ~[?:?]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:652) ~[?:?]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562) ~[?:?]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997) ~[?:?]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[?:?]
	at java.lang.Thread.run(Thread.java:1583) ~[?:?]
```

--- a/docs/changelog/119296.yaml
+++ b/docs/changelog/119296.yaml
@@ -1,0 +1,6 @@
+pr: 119296
+summary: Fix writing for LOOKUP status
+area: ES|QL
+type: bug
+issues:
+ - 119086

--- a/x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/plugin/EsqlPlugin.java
+++ b/x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/plugin/EsqlPlugin.java
@@ -61,6 +61,7 @@ import org.elasticsearch.xpack.esql.action.RestEsqlDeleteAsyncResultAction;
 import org.elasticsearch.xpack.esql.action.RestEsqlGetAsyncResultAction;
 import org.elasticsearch.xpack.esql.action.RestEsqlQueryAction;
 import org.elasticsearch.xpack.esql.enrich.EnrichLookupOperator;
+import org.elasticsearch.xpack.esql.enrich.LookupFromIndexOperator;
 import org.elasticsearch.xpack.esql.execution.PlanExecutor;
 import org.elasticsearch.xpack.esql.expression.ExpressionWritables;
 import org.elasticsearch.xpack.esql.plan.PlanWritables;
@@ -192,6 +193,7 @@ public class EsqlPlugin extends Plugin implements ActionPlugin {
         entries.add(SingleValueQuery.ENTRY);
         entries.add(AsyncOperator.Status.ENTRY);
         entries.add(EnrichLookupOperator.Status.ENTRY);
+        entries.add(LookupFromIndexOperator.Status.ENTRY);
 
         entries.addAll(BlockWritables.getNamedWriteables());
         entries.addAll(ExpressionWritables.getNamedWriteables());

